src/server/assets/debuggerWorker.js

```diff
@@ -30,19 +34,33 @@ onmessage = (function() {
     };
   })();
 
+  const processEnqueuedMessages = function() {
+    while (messageQueue.length) {
+      const messageProcess = messageQueue.shift();
+      messageProcess();
+    }
+    shouldQueueMessages = false;
+  };
+
   const messageHandlers = {
     executeApplicationScript(message, sendReply) {
       for (const key in message.inject) {
         self[key] = JSON.parse(message.inject[key]);
       }
 
+      shouldQueueMessages = true;
+
       function evalJS(js) {
         try {
-          eval(js);
+          eval(
+            js
+              .replace(/this\[“webpackHotUpdate”\]/g, 'self[“webpackHotUpdate”]')
```

Comment:
Maybe I don't have enough context, but this quote `“` is different than default double quote:`"`. Is this intended?